backend/src/main/java/heartbeat/controller/report/dto/response/CycleTime.java

```diff
@@ -0,0 +1,24 @@
+package heartbeat.controller.report.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CycleTime {
+
+	private double totalTime;
+
+	private double averageCircleTimePerCard;
+
+	private double averageCycleTimePerSP;
+
+	private List<CycleTimeForSelectedStepItem> cycleTimeForSelectedStepsList;
```

Comment:
cycleTimeForSelectedStepsList to cycleTimeForSelectedSteps?

Reply:
done